docs/chain-abstraction/chain-signatures/implementation.md

```diff
@@ -189,24 +200,35 @@ To construct the transaction to be signed use the method `prepareTransactionForS
 
 Once the transaction is created and ready to be signed, a signature request is made by calling `sign` on the MPC smart contract.
 
-The method requires three parameters:
+The method requires four parameters:
 
-  1. The `payload` (or hash) to be signed for the target blockchain
+  1. The `payloads` (or hashes) to be signed for the target blockchain
   2. The derivation `path` for the account we want to use to sign the transaction
-  3. The `key_version`, `0` for `Secp256k1` signatures and `1` for `Ed25519` signatures.
+  3. The `keyType`, `Ecdsa` for `Secp256k1` signatures and `Eddsa` for `Ed25519` signatures.
+  4. The `signerAccount` which contains the `accountId` that is signing and the `signAndSendTransactions` function from the [wallet selector](../../tools/wallet-selector.md).
+
+  <details>
+  <summary> Signing with a private key </summary>
```

Comment:
I think we can safely remove this, since these are not docs for the library, but for the example